snuba/datasets/entities/sessions.py

```diff
@@ -266,3 +270,43 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             ),
             ProjectRateLimiterProcessor(project_column="project_id"),
         ]
+
+
+class OrgSessionsEntity(Entity):
+    def __init__(self) -> None:
+        storage = get_storage(StorageKey.SESSIONS_HOURLY)
+
+        super().__init__(
+            storages=[storage],
+            query_pipeline_builder=SimplePipelineBuilder(
+                query_plan_builder=SingleStorageQueryPlanBuilder(storage=storage)
+            ),
+            abstract_column_set=ColumnSet(
+                [
+                    ("org_id", UInt(64)),
+                    ("project_id", UInt(64)),
+                    ("started", DateTime()),
+                ]
+            ),
+            join_relationships={},
+            writable_storage=None,
+            validators=None,
+            required_time_column="started",
+        )
+
+    def get_extensions(self) -> Mapping[str, QueryExtension]:
+        return {
+            "timeseries": TimeSeriesExtension(
```

Comment:
Extensions are only used by the legacy syntax. Since you'll be using SnQL you can remove this.